helm/thingsboard/templates/coap-transport.yaml

```diff
@@ -62,6 +62,18 @@ spec:
             value: "tb-coap-transport"
           - name: TB_QUEUE_TYPE
             value: "kafka"
+          {{- if eq .Values.monitoring.enabled true }}
+          - name: METRICS_ENABLED
+            value: "true"
+          - name: METRICS_ENDPOINTS_EXPOSE
```

Comment:
this expose is related to this config: https://github.com/thingsboard/thingsboard/blob/07e834bee063b2f55dfdd65c58ad360406637409/application/src/main/resources/thingsboard.yml#L1214

I think here you need to add "prometheus" instead of true